hs_core/testing.py

```diff
@@ -180,14 +168,19 @@ def resource_file_oprs(self):
         if res.resource_federation_path:
             fed_test_file1_full_path = '/{zone}/home/{uname}/{fname}'.format(
                 zone=settings.HS_USER_IRODS_ZONE, uname=user.username, fname=file_name_list[0])
-            add_resource_files(res.short_id, fed_res_file_names=[fed_test_file1_full_path],
-                               fed_copy_or_move='copy',
-                               fed_zone_home_path=res.resource_federation_path)
+            # TODO: why isn't this a method of resource?
```

Comment:
Agreed.  This should be implemented a function of the BaseResource class. May be in a follow up PR one of us can do some additional internal API rework.